src/components/dynamic/admin/services/judging/Table.jsx

```diff
@@ -41,14 +41,20 @@ const Table = ({ data }) => {
                 <div key={index} className="flex items-center my-2">
                   <p className="font-semibold mb-0 mr-2">{index + 1}</p>
                   <div className="flex items-center">
-                    {judges.map(({ name, affiliation }, i) => (
-                      <Tag
-                        classes="mx-1"
-                        color={COLORS[affiliation]}
-                        key={i}
-                        text={name}
-                      />
-                    ))}
+                    {judges.map(({ name, table, affiliation }, i) => {
```

Comment:
we should be able to keep all the code in the toolbar and do the concat within the toolbar itself instead of passing an extra field